src/styles/_stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  width: 97px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars__star {
+  background-image: url("../images/star.svg");
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  width: 13px;
+  height: 12px;
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
```

Comment:
reuse solution from `Stars` task. So we need to apply styles for all modifiers

Reply:
still not resolved, you need to use ALL modifiers from previous task `stars`, not only for 4 stars, 1,2,3,4,5 should be able to use
![image](https://github.com/mate-academy/layout_product-cards/assets/55200530/47ee06eb-fedc-4fac-814d-431eebad7408)